config/config.go

```diff
@@ -157,6 +158,14 @@ type AuctionTimeouts struct {
 	Max uint64 `mapstructure:"max"`
 }
 
+func (pf *AccountPriceFloors) validate(errs []error) []error {
+
+	if pf.EnforceFloorRate < 0 || pf.EnforceFloorRate > 100 {
+		errs = append(errs, fmt.Errorf(`account_defaults.price_floors.enforce_floors_rate should be between 0 and 100`))
+	}
+	return errs
+}
+
```

Comment:
Actually can we move this function into `config/accounts.go` right after the `AccountPriceFloors` struct definition so we keep the receiver methods close their data?

Reply:
moved into `config/account.go`